### PR TITLE
[RHELC-1267] Add Alma, Rocky 8.9 to pipeline, 8.8 as EUS

### PR DESCRIPTION
**Pull request by danmyway**
* add published 8.9 AWS AMIs to the pipeline
* keep 8.8 as EUS
* modify the playbooks to hardcode 8.8 repos
* address changes in metadata

Jira Issues: [RHELC-1267](https://issues.redhat.com/browse/RHELC-1267)

Checklist

- [x] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [x] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [x] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

**Comment by kokesak** [comment]
Please remove also this line (OL8.8 is not the latest anymore):
https://github.com/oamg/convert2rhel/blob/main/tests/integration/common/checks-after-conversion/test_release_version.py#L25

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -551,3 +551,28 @@ def hybrid_rocky_image(shell, system_release):
...
+    yum_command = f"yum list installed --disablerepo=* --enablerepo={reponame} | awk '$3==\"@{reponame}\" {{print $1}}'"
+
+    # Get list of installed packages from the rcmtools repository
+    shell(f"{yum_command} > {rcmtools_pkgs}")
+
+    # Remove the installed packages
+    with open(rcmtools_pkgs, "r") as file:
+        for line in file:
+            package = line.strip()
+            shell(f"yum remove --quiet -y {package}")
```
[the remark this answers is not in the document]
Comment (danmyway) [reply tests/integration/conftest.py:575]: Makes sense, thanks!

**Pull request merged**